### PR TITLE
Triplelift adapter tdid support

### PR DESCRIPTION
**Pull request by rmcnierney**
## Type of change
- [x] Feature

## Description of change
Updating the TripleLift adapter to read The Trade Desk's Unified ID - `bidRequest.userId.tdid`

### CODE REVIEW

**Comment by rmcnierney** [comment]
Thanks, @jaiminpanchal27. package-lock.json should now be removed

**Comment by jaiminpanchal27** [comment]
@rmcnierney Please create a docs PR to update the list here http://prebid.org/dev-docs/modules/userId.html#adapters-supporting-the-user-id-sub-modules

**Comment by rmcnierney** [comment]
@jaiminpanchal27 docs PR here - [Update docs for Triplelift tdid support #1392](https://github.com/prebid/prebid.github.io/pull/1392)

File: modules/tripleliftBidAdapter.js [hunk 1]
```diff
@@ -91,6 +91,13 @@ function _buildPostBody(bidRequests) {
     }
   });
 
+  let eids = handleConsortiaUserIds(bidderRequest)
```
Comment (cdsmith16) [inline modules/tripleliftBidAdapter.js]: add semicolon

**Pull request merged**